src/app/pages/common/info-dialog/info-dialog.component.scss

```diff
@@ -1,5 +1,12 @@
 .info-panel {
     max-height: 115px !important;
+
+    span {
+      display: block;
+      unicode-bidi: embed;
+      font-family: monospace;
+      white-space: pre;
```

Comment:
Set the white-space to normal

Reply:
This was changed intentionally to preserve white spaces in a string. Let me show a difference